runtime/configs/proto/fisherman_config.proto

```diff
@@ -0,0 +1,14 @@
+
+syntax = "proto3";
+
+package configs;
+
+option go_package = "github.com/pokt-network/pocket/runtime/configs";
+
+// FishermanConfig defines the configuration for the node acting as a fisherman. Fishermen grade and enforce the quality of the Web3 access provided by Servicers
+// TODO(0xbigboss): Add fields
```

Comment:
If you can't think of it right now, let's remove the TODOs/INVESTIGATE and add things as we implement e2e features. Ditto for the other configs.

I'm working on those in a notion doc for now

<img width="583" alt="Screenshot 2023-05-03 at 7 36 52 PM" src="https://user-images.githubusercontent.com/1892194/236098709-d41c1840-1153-4d16-b35a-8dd1527307b9.png">